### ISSUE TITLE
JAWS reads filter bar which is invisible

### ISSUE THREAD

**Issue by xiepingp** [carbon-design-system/carbon#9343]
## Environment

> Operating system

Windows

> Browser

Latest Chrome

> Automated testing tool and ruleset

IBM CI 162 Checkpoint Violation: IBM 1.3.2 Meaningful Sequence, IBM 2.4.3 Focus Order
IBM Guidance: https://www.ibm.com/able/requirements/requirements/#1_3_2
https://www.ibm.com/able/requirements/requirements/#2_4_3
Test Tool Type: JAWS Screen Reader 2021

> Assistive technology used to verify

## Detailed description

When checkbox is selected in Datatable with Batch actions, the batch action bar is present and the filter bar is hidden, then I press **Up arrow** to move the focus back to the batch action bar, Jaws reads the filter bar content before the batch actions which is not expected, the hidden filter bar should not be read by jaws in this case.

> What version of the Carbon Design System are you using?

carbon-components-react@7.40.0

> What did you expect to happen?

The hidden filter bar should not be read by jaws in this case.

> What happened instead?

Hidden filter bar read by JAWS when not displayed.

> What WCAG 2.1 checkpoint does the issue violate?

IBM CI 162 Checkpoint Violation: IBM 1.3.2 Meaningful Sequence, IBM 2.4.3 Focus Order

## Steps to reproduce the issue

Use JAWS to read page content. 
1. move the focus to checkbox in the datatable then select it
2. press **Up arrow** to move the focus back to the batch action bar

> Please create a reduced test case in CodeSandbox

Use storybook here:

https://react.carbondesignsystem.com/?path=/story/components-datatable-batch-actions--usage

## Additional information

![invisible - carbon](https://user-images.githubusercontent.com/63960906/127259618-6f15baed-0fcf-458d-86cc-5e179603c206.png)

**Comment by mashenka123** [comment]
@tw15egan Could you please confirm when this would be fixed? This is blocking Schematics signoff, thank you!!

**Comment by joshblack** [comment]
Sorry for the delay @xiepingp, TJ has been out on leave but I'm tackling this issue this sprint.

When diving into it, I believe that the behavior of hiding the bar is one that can be addressed on the application side. I'll open a PR with what I think the fix is (adding `aria-hidden` in this case) and would love to hear if this example would help out with this use-case.

Thanks again for being so patient, I apologize for the delay on our end for this!

**Comment by xiepingp** [comment]
@finken2, could you please take a look?

**Comment by xiepingp** [comment]
Thanks for the fixing @joshblack !! The invibile bar now is not read by jaws! 👍👍
![carbon 9343 - fixed](https://user-images.githubusercontent.com/63960906/145132284-fc38fe6b-3bec-4f3e-a757-8a17b6d36bdf.png)
